patroni/ctl.py

```diff
@@ -1299,10 +1299,13 @@ def version(obj, cluster_name, member_names):
 def history(obj, cluster_name, fmt):
     cluster = get_dcs(obj, cluster_name).get_cluster()
     history = cluster.history and cluster.history.lines or []
+    table_header_row = ['TL', 'LSN', 'Reason', 'Timestamp', 'Master']
```

Comment:
Lets better call it Leader, which is in fact the New Leader, because it shows name of the leader on the new timeline.
```suggestion
    table_header_row = ['TL', 'LSN', 'Reason', 'Timestamp', 'New Leader']
```

Also, please keep in mind that information about the who was the leader on timeline=1 is lost forewer.

Reply:
fine this is nice